File: app/back/src/main/java/net/seijishikin/jp/normalize/manage/kanrensha/dto/wktbl_std/UpdateWkTblStdKigyouDtCapsuleDto.java
package net.seijishikin.jp.normalize.manage.kanrensha.dto.wktbl_std;

import java.io.Serializable;

import net.seijishikin.jp.normalize.common_tool.dto.FrameworkCapsuleDto;
import net.seijishikin.jp.normalize.manage.kanrensha.entity.WkTblKanrenshaKigyouDtMasterEntity;

/**
 * 企業／団体一括登録マスタ標準ワークテーブル更新CapsuleDto
 */
public class UpdateWkTblStdKigyouDtCapsuleDto extends FrameworkCapsuleDto // NOPMD DataClass
        implements Serializable {

    /** Serialize id */
    private static final long serialVersionUID = 1L;

    /** 編集対象Entity */
    private WkTblKanrenshaKigyouDtMasterEntity wkTblKanrenshaKigyouDtMasterEntity = new WkTblKanrenshaKigyouDtMasterEntity();

    /**
     * 編集対象Entityを取得する
     * 
     * @return 編集対象Entity
     */
    public WkTblKanrenshaKigyouDtMasterEntity getWkTblKanrenshaKigyouDtMasterEntity() {
        return wkTblKanrenshaKigyouDtMasterEntity;
    }

    /**
     * 編集対象Entityを設定する
     * 
     * @param wkTblKanrenshaKigyouDtMasterEntity 編集対象Entity
     */
    public void setWkTblKanrenshaKigyouDtMasterEntity(
            final WkTblKanrenshaKigyouDtMasterEntity wkTblKanrenshaKigyouDtMasterEntity) {
        this.wkTblKanrenshaKigyouDtMasterEntity = wkTblKanrenshaKigyouDtMasterEntity;
    }

}

File: app/front/src/components/main/dto/wktbl_std/updateWkTblStdKigyouDtCapsuleDto.ts
import { WkTblKanrenshaKigyouDtMasterEntity, type WkTblKanrenshaKigyouDtMasterEntityInterface } from "../../entity/wkTblKanrenshaKigyouDtMasterEntity";
import { type FrameworkCapsuleDtoInterface, FrameworkCapsuleDto } from "seijishikin-jp-normalize_common-tool";

interface UpdateWkTblStdKigyouDtCapsuleDtoInterface extends FrameworkCapsuleDtoInterface {

    /** 編集対象Entity */
    wkTblKanrenshaKigyouDtMasterEntity: WkTblKanrenshaKigyouDtMasterEntityInterface;

}

/**
 * 企業／団体一括登録マスタ標準ワークテーブル更新CapsuleDto
 */
class UpdateWkTblStdKigyouDtCapsuleDto extends FrameworkCapsuleDto
    implements UpdateWkTblStdKigyouDtCapsuleDtoInterface {

    /** 編集対象Entity */
    wkTblKanrenshaKigyouDtMasterEntity: WkTblKanrenshaKigyouDtMasterEntityInterface;

    constructor() {
        super();
        this.wkTblKanrenshaKigyouDtMasterEntity = new WkTblKanrenshaKigyouDtMasterEntity();
    }
}

export { type UpdateWkTblStdKigyouDtCapsuleDtoInterface, UpdateWkTblStdKigyouDtCapsuleDto }
